server/src/api/community.rs

```diff
@@ -203,13 +203,10 @@ impl Perform for Oper<GetCommunity> {
           id,
         });
       }
-
-      // TODO
-      1
-    // let fut = async {
-    //   ws.chatserver.send(GetCommunityUsersOnline {community_id}).await.unwrap()
-    // };
-    // Runtime::new().unwrap().block_on(fut)
+      ws.chatserver
+        .send(GetCommunityUsersOnline { community_id })
+        .await
+        .unwrap_or(1)
```

Comment:
Is there a particular reason why you are returning 1 instead of logging an error?

Reply:
Ya, I don't want the whole API method to fail if websocket stuff doesn't get sent / received for whatever reason... its not vital.